_data/projects/search/releases/7.2/series.yml

```diff
@@ -1,10 +1,13 @@
 summary: >-
-  Enable query string predicates on numeric/date fields,
+  Switch project license to Apache License 2.0,
+  enable query string predicates on numeric/date fields,
   query parameters,
   document tree projection for the Lucene backend,
+  accept multiple ranges in a range predicate,
```

Comment:
Not sure I'd mention all such DSL improvements in here as it can lead to a big wall of text -- see 7.0 here:

![image](https://github.com/hibernate/hibernate.org/assets/412878/7dd72be6-54f5-4635-b3b2-e75d60c1b336)

For now it's probably still bearable, but I wanted to draw your attention on this :) Do better than what I did in the past xD

Reply:
🙈 good point 😃 
Thanks for the suggestion, I've removed the line about the range predicate